pandas/core/generic.py

```diff
@@ -2776,15 +2776,19 @@ def to_latex(self, buf=None, columns=None, col_space=None, header=True,
                  sparsify=None, index_names=True, bold_rows=False,
                  column_format=None, longtable=None, escape=None,
                  encoding=None, decimal='.', multicolumn=None,
-                 multicolumn_format=None, multirow=None):
+                 multicolumn_format=None, multirow=None, caption=None,
+                 label=None):
         r"""
-        Render an object to a LaTeX tabular environment table.
+        Render object to a LaTeX tabular, longtable, or nested table/tabular.
```

Comment:
``longtable`` has its own argument. The fact (of which I'm not a fan) that the table/tabular environment instead can only be activated by passing a ``caption`` or a ``label`` is not stated anywhere in the docstring: it should be clarified.

Reply:
@toobaz the table/tabular environment will be used when `longtable` is `False` which is the current behavior.

Reply:
Yes, my observation was about the nested table/tabular. When is it created?

Reply:
Or more generally: can I have a nested table/tabular _without_ passing a caption or a label?